content/docs/performance_scale/perf-scale.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Open Service Mesh Performance and Scalability"
```

Comment:
Change to just "OSM Performance and Scalability"

Reply:
I'm going to move this doc under this section: https://release-v1-0.docs.openservicemesh.io/docs/guides/ha_scale/

I think we just keep "Performance and Scalability", similar to the other doc in that section. WDYT?